src/scout_apm/falcon.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from scout_apm.api import install
+from scout_apm.core.ignore import ignore_path
+from scout_apm.core.tracked_request import TrackedRequest
+
+try:
+    import falcon
+except ImportError:  # pragma: no cover
+    falcon = None
```

Comment:
Is this used?

Reply:
I just did this so that the module would remain importable when falcon isn't installed, but I guess it's not needed since none of the other library integrations use this pattern.